### PR TITLE
Add @nextcloud/event-bus to babel transpile

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

**Comment by skjnldsv** [comment]
Jesus, I forced push the wrong repo --"

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "@nextcloud/password-confirmation": "^1.0.1",
     "@nextcloud/paths": "^1.1.2",
     "@nextcloud/router": "^1.1.0",
-    "@nextcloud/vue": "^2.3.0",
+    "@nextcloud/vue": "^1.5.0",
```
Comment (nickvergessen) [inline package.json:41]: ?

Comment (skjnldsv) [reply package.json:41]: https://github.com/nextcloud/server/pull/22074#issuecomment-667074858

**Review by nickvergessen — APPROVED** [review]

**Review by georgehrke — APPROVED** [review]
Changes look fine.
Didn't check in IE11 though

**Pull request merged**